README.md

```diff
@@ -384,3 +386,48 @@ const assert = v.assert(v.range([-10, 10]));
 assert(4); // pass
 assert(-100); // fail
 ```
+
+
+## Custom validators
+One of the primary goals of Fusspot is to be customizable out of the box. There are multiple ways to which one can create a custom validator. 
+
+### Simple
+A simple custom validator is nothing but a function which accepts the input `value` and returns a `string` if the input `value` doesn't pass the test. The `string` returned would then be used to show a helpful error to users. Below is an example of a path validator for node environment.
+
+```javascript
+const path = require('path');
+
+function validateAbsolutePaths(value) {
+  if (typeof value !== 'string' || !path.isAbsolute(value)) {
+    return 'absolute path';
+  }
+}
+
+const assert = v.assert(validateAbsolutePaths);
+assert('../Users'); // fail
+// Error: value must be an absolute path.
+assert('/Users'); // pass
+```
+
+### Customizing the entire error message 
+To customize the entire error message, a validator can return a function. This function gets called with an object containing
+ - **path**: This is an array containing the location error. If an error is in 
+ - **apiName**: The value of [options.apiName](#vassertrootvalidator-options) passed to the assert which would be running this validator.
```

Comment:
I don't think I understand this parameter. In the example below it's `undefined`, so that doesn't help me. Could you explain more about it?

Reply:
Whoops, I have removed the parameter since it seems confusing. Basically it is used by all validators to prepend all the error messages.